work-order-service/src/main/java/org/mkgroup/zaga/workorderservice/controller/WorkOrderController.java

```diff
@@ -29,16 +39,45 @@
 	@Autowired
 	WorkOrderRepository wrepo;
 	
+	@Autowired
+	SAPGatewayProxy gwProxy;
+	
+	@Autowired
+	SAP4HanaProxy sap4hana;
+	
+	@Autowired
+	SAPAuthConfiguration authConfiguration;
+	
 	@PostMapping("/createWorkOrder")
-	public ResponseEntity<?> createWorkOrder(@RequestBody WorkOrderDTO request){
+	public ResponseEntity<?> createWorkOrder(@RequestBody WorkOrderDTO request) throws Exception{
```

Comment:
Not quietly what I had in mind. Function addWorkOrder can couse many exception. Try to introduce ResponseEntityExceptionHandler  which can help handling many types of exception, custom or default and set a exception throwing standard for whole team. This is top priority :fire: